src/components/FAQ.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+const FAQSection = () => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
+
+  const toggleAccordion = (index: number) => {
+    setActiveIndex(index === activeIndex ? null : index);
+  };
+
+  const faqs = [
+    {
+      question: 'What is Suraksha?',
+      answer: 'Suraksha is a platform designed to ensure women’s safety by providing real-time tracking, safest routes, and emergency alerts.',
+    },
+    {
+      question: 'How does live location tracking work?',
+      answer: 'Live location tracking uses GPS to share your real-time position with your emergency contacts, allowing them to monitor your safety.',
+    },
+    {
+      question: 'Can I mark unsafe areas?',
+      answer: 'Yes, users can report and mark areas as safe or unsafe, which helps others plan safer routes.',
+    },
+    {
+      question: 'How can I contribute to Suraksha?',
+      answer: 'You can contribute by providing feedback, sharing your experience, and donating through the platform to support further development.',
+    },
+  ];
+
+  return (
+    <section
+      id="faq"
+      className="py-12 px-4 sm:px-6 md:px-8 lg:px-10 bg-gradient-to-t from-[#e0f2ff] to-[#a0c8f0] text-black"
+    >
+      <div className="flex flex-col items-center max-w-5xl mx-auto">
+        <div className="text-3xl sm:text-4xl md:text-5xl lg:text-6xl pt-6 font-bold tracking-tighter mb-8 text-center bg-gradient-to-b from-black to-[#6a0dad] text-transparent bg-clip-text">
+          Frequently Asked Questions
+        </div>
+
+        <div className="w-full space-y-4">
+          {faqs.map((faq, index) => (
+            <div
```

Comment:
Weren't you going to use accordion for this?

Reply:
material UI ka accordion component. Its your choice whatever is comfortable with you